docs/packages/markdown/parseMarkdown.js

```diff
@@ -249,6 +249,9 @@ function createRender(context) {
         `<a aria-labelledby="${hash}" class="anchor-link-style" href="#${hash}" tabindex="-1">`,
         '<svg><use xlink:href="#anchor-link-icon" /></svg>',
         '</a>',
+        `<button class="comment-link-style" data-feedback-hash="${hash}">`,
```

Comment:
This is not accessible, see https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh reports:

<img width="426" alt="Screenshot 2022-10-26 at 14 02 26" src="https://user-images.githubusercontent.com/3165635/198021256-4af051fb-6acc-4d7f-9074-b8044198c46d.png">